pages/coinbase/homepage.py

```diff
@@ -30,27 +40,242 @@ def __init__(self, driver, spawn=True):
     company_menu_loc =      (by.XPATH, "//button[contains(text(),'Company')]")
     earn_crypto_link_loc =  (by.XPATH, "//a[@title='Earn crypto']")
     sign_in_link_loc =      (by.XPATH, "//a[@title='Sign in']")
-    header_get_started_button_loc =(
-        by.XPATH, "//a[.='Get started']")
+    header_get_started_button_loc = (by.XPATH, "//a[.='Get started']")
+
+         
+    # CREATE ACCOUNT
+
+    email_address_field_loc = (by.XPATH, "//input[@type='email']")
+    get_started_button_loc =  (by.XPATH,
+                            "//input[@type='email']/following-sibling::button")
+
+    # CRYPTO CURRENCY MENU
+
+    bitcoin_text_link_loc =    (by.XPATH, "//h4[.='Bitcoin']")
+    bitcoin_BTC_link_loc =     (by.XPATH, "//h4[.='BTC']")
+    bitcoin_logo_link_loc =    (by.XPATH, "//img[@alt='Bitcoin logo']")
+    bitcoin_buy_button_loc =   (by.XPATH, "//a[@href='/price/bitcoin']/button")
+
+    ethereum_text_link_loc =  (by.XPATH, "//h4[.='Ethereum']")
+    ethereum_ETH_link_loc =   (by.XPATH, "//h4[.='ETH']") 
+    ethereum_logo_link_loc =  (by.XPATH, "//img[@alt='Ethereum logo']")
+    ethereum_buy_button_loc=  (by.XPATH, "//a[@href='/price/ethereum']/button")
+
+    bitcoinCash_text_link_loc =   (by.XPATH, "//h4[.='Bitcoin Cash']")        
+    bitcoinCash_BCH_link_loc =    (by.XPATH, "//h4[.='BCH']")
+    bitcoinCash_logo_link_loc =   (by.XPATH, "//img[@alt='Bitcoin Cash logo']")
+    bitcoinCash_buy_button_loc =  (by.XPATH,
+                                   "//a[@href='/price/bitcoin-cash']/button")
+
+    litecoin_text_link_loc =  (by.XPATH, "//h4[.='Litecoin']")
+    litecoin_LTC_link_loc =   (by.XPATH, "//h4[.='LTC']")
+    litecoin_logo_link_loc =  (by.XPATH, "//img[@alt='Litecoin logo']")
+    litecoin_buy_button_loc=  (by.XPATH, "//a[@href='/price/litecoin']/button")
+
+    # EARNING LINKS BLCOK
+
+    start_earning_button_loc = (by.XPATH,
+                               "//button[contains(@class,'EarnUpsellBanner')]")
+    footer_start_earning_button_loc = (by.XPATH,
+                               "//button[contains(@class,'EarnFooterBanner')]")
+    maker_link_loc =          (by.XPATH, "//a[@to='/earn/maker']")
+    celo_link_loc =           (by.XPATH, "//a[@to='/earn/celo']")  
+    compound_link_loc =       (by.XPATH, "//a[@to='/earn/compound']") 
+    EOS_link_loc =            (by.XPATH, "//a[@to='/earn/eos']")
+    view_more_link_loc =      (by.XPATH, "//a[@to='/earn']")
+
+    # MOBILE APPS
+    
+    android_app_link_loc =        (by.XPATH, "//a[contains(@href,'android')]")
+    ios_app_link_loc =            (by.XPATH, "//a[.='iOS']")
+
+    # COINBASE SECURITY/INSURANCE
+
+    secure_storage_link_loc =       (by.XPATH,
```

Comment:
please fix the spacing

I'm wondering if it looks different in your IDE. If so you should probably do something about that.